cmd/juju/action/run_test.go

```diff
@@ -216,6 +220,10 @@ func (s *RunSuite) TestInit(c *gc.C) {
 		expectUnits:  []string{"mysql/leader"},
 		expectAction: "valid-action-name",
 		expectKVArgs: [][]string{},
+	}, {
```

Comment:
This is really the same as the other newly added test above. What we could add is a second test with "app1/leader" and "app2/2" and expect the error. If we do that, could we move this up to where the other test is so that they are together.

Reply:
yep complete mistake on my part.  Done.